packages/component-login/client/index.js

```diff
@@ -68,7 +67,8 @@ class LoginPage extends React.Component {
         LoginPage.setToken(null)
         // TODO expose this error once we have a UI to do so
         this.props.history.push('/login', { error: err.message })
-        logger.error(err)
+        // eslint-disable-next-line no-console
+        console.error(err)
```

Comment:
Why has this been reverted @SemraSh ?

Reply:
Because `logger` is available only in the server and it is not going to be safe to make it public.